test_cdk_pgstac_infra.py

```diff
@@ -0,0 +1,175 @@
+    "bbox":[-78.40290984426046,51.07724585591961,-77.04127077089376,51.92130718597872],
+    "assets":{"csv":{"href":"s3://nasa-maap-data-store/file-staging/nasa-map/icesat2-boreal/boreal_agb_202302031675450345_0177_train_data.csv","type":"text/csv","roles":["data"],"title":"CSV","description":"CSV of training data"},"tif":{"href":"s3://nasa-maap-data-store/file-staging/nasa-map/icesat2-boreal/boreal_agb_202302031675450345_0177.tif","type":"image/tiff; application=geotiff; profile=cloud-optimized","roles":["data"],"title":"Cloud Optimized GeoTIFF of boreal data","description":"Cloud Optimized GeoTIFF of boreal data","raster:bands":[{"scale":1.0,"nodata":-9999.0,"offset":0.0,"sampling":"area","data_type":"float32","histogram":{"max":105.78067016601562,"min":7.345508575439453,"count":11,"buckets":[2194,2380,972,469,298,184,85,30,8,4]},"statistics":{"mean":25.281058933423914,"stddev":13.868902983070951,"maximum":105.78067016601562,"minimum":7.345508575439453,"valid_percent":0.6317138671875}},{"scale":1.0,"nodata":-9999.0,"offset":0.0,"sampling":"area","data_type":"float32","histogram":{"max":60.75077819824219,"min":1.4587666988372803,"count":11,"buckets":[5140,1167,250,42,14,4,3,1,2,1]},"statistics":{"mean":5.982097533589976,"stddev":3.7930746502586974,"maximum":60.75077819824219,"minimum":1.4587666988372803,"valid_percent":0.6317138671875}}]}},
+    "properties":{"datetime":"2023-02-15T00:00:00+00:00","proj:bbox":[4598521.999999994,5643304.000000009,4688521.999999994,5733304.000000009],"proj:shape":[3000,3000],"proj:geometry":{"type":"Polygon","coordinates":[[[4598521.999999994,5643304.000000009],[4688521.999999994,5643304.000000009],[4688521.999999994,5733304.000000009],[4598521.999999994,5733304.000000009],[4598521.999999994,5643304.000000009]]]},"proj:transform":[30.0,0.0,4598521.999999994,0.0,-30.0,5733304.000000009,0.0,0.0,1.0]},
+    "stac_extensions":["https://stac-extensions.github.io/projection/v1.1.0/schema.json","https://stac-extensions.github.io/raster/v1.1.0/schema.json"],
+    "geometry":{"type":"Polygon","coordinates":[[[14.114837413118664,67.218607039971],[12.438229073696998,67.70894310918132],[11.17860397724852,67.06631312684836],[12.837754767891637,66.58867761064732],[14.114837413118664,67.218607039971]]]}
+}
+
+
+#Validating the test collection
+try:
+    pystac.validation.validate_dict(test_collection)
+    print("Valid test collection")
+except:
+    raise STACValidationError
+
+#Validating the test item
+try:
+    pystac.validation.validate_dict(test_item)
+    print("Valid test item")
+except:
+    raise STACValidationError
+
+#Authentication - Get secret value
+stage = "dev"
+stac_register_service_id = "MAAP-workflows-EsykqB"
```

Comment:
this isn't correct I believe. The `secret_id` variable should be assigned the value `"MAAP-STAC-auth-dev/MAAP-workflows"`. 

So I'd replace 

`stac_register_service_id = "MAAP-workflows-EsykqB"`

with

`stac_register_service_id="MAAP-workflows"`